release notes/v0.44.0.md

```diff
@@ -0,0 +1,224 @@
+- [#2920](https://github.com/grafana/k6/pull/2920). Updated Go version that we use for k6 compilation to 1.20.x. The docker image is also updated to use alpine:3.17.
+- [#2986](https://github.com/grafana/k6/pull/2986). Refactored goja's `isNullish` to be a part of `js/common`.
+- [#2960](https://github.com/grafana/k6/pull/2960). Refactored `sirupsen/logrus` usage.
+- [#2999](https://github.com/grafana/k6/pull/2999). Directly embed lib.TestPreInitState in js/common.InitEnvironment.
+- [#2931](https://github.com/grafana/k6/pull/2931), [#2940](https://github.com/grafana/k6/pull/2940), [#2895](https://github.com/grafana/k6/pull/2895), [#3002](https://github.com/grafana/k6/pull/3002). Updated k6's dependencies. Added `goja`'s generator support.
+- [#2947](https://github.com/grafana/k6/pull/2947), [#2943](https://github.com/grafana/k6/pull/2943), [#2946](https://github.com/grafana/k6/pull/2946), [#3009](https://github.com/grafana/k6/pull/3009), [#3012](https://github.com/grafana/k6/pull/3012), [#2894](https://github.com/grafana/k6/pull/2894). Tests' fixes, refactoring, and improvements.
+- [#2891](https://github.com/grafana/k6/pull/2891), [#2921](https://github.com/grafana/k6/pull/2921), [#2923](https://github.com/grafana/k6/pull/2923), [#2990](https://github.com/grafana/k6/pull/2990), [#2995](https://github.com/grafana/k6/pull/2995), [#3016](https://github.com/grafana/k6/pull/3016), [#2989](https://github.com/grafana/k6/pull/2989). Linters and formatting fixes.
+- [#3005](https://github.com/grafana/k6/pull/3005). A samples catalog was renamed to examples.
+- [xk6-browser#776](https://github.com/grafana/xk6-browser/pull/776) Fix test for preset flags for chrome on darwin based systems.
+- [xk6-browser#782](https://github.com/grafana/xk6-browser/pull/782) Refactor mapping of "special" functions to work with a `map`.
+- [xk6-browser#783](https://github.com/grafana/xk6-browser/pull/783) Fix mapping by unmapping internal methods.
+- [xk6-browser#797](https://github.com/grafana/xk6-browser/pull/797) Fix for multi browser close vu was missing in Context.
+- [xk6-browser#796](https://github.com/grafana/xk6-browser/pull/796) Return an error from `browserContext.SetExtraHTTPHeaders` and handle panic in mapping layer.
+- [xk6-browser#798](https://github.com/grafana/xk6-browser/pull/798) Add locator mapping test.
+- [xk6-browser#799](https://github.com/grafana/xk6-browser/pull/799) Add a testing functionality to detect redundant mappings.
+- [xk6-browser#802](https://github.com/grafana/xk6-browser/pull/802) Add assert log contains and log dump helpers.
+- [xk6-browser#807](https://github.com/grafana/xk6-browser/pull/807) Fix incorrect keyboard key code on up/down key press.
+- [xk6-browser#810](https://github.com/grafana/xk6-browser/pull/810) Rename ErrInternal to ErrFatal and use abort/interrupt.
+- [xk6-browser#821](https://github.com/grafana/xk6-browser/pull/821) Upgrade to k6 v0.43.1.
```

Comment:
@ankur22 it's probably redundant, or we could put this under the `Updated k6's dependencies.` (maybe just adjust naming to `Updated dependencies.`)

Reply:
I've made the change: https://github.com/grafana/k6/commit/458dfb713fe0361eb275846c4feebc3e3b24b9eb